Changes implemented

ShurionUA opened the pull request:
README.md created!
Bug fixed.

ShurionUA committed:
README.md created!
Bug fixed.

Nick97-git reviewed (commented):
README:
- Remove `It uses Hibernate and Spring`
- `users(customers)` -> `users`
- After `List of endpoints` on the next line write something like this `Depending on the roles user will have access to different endpoints: `
- `before authentication` -> `Without roles`
-  every next block start with an upper case letter
- let's remove block `Requirements`
- `Configure before launch`, let's add two steps before the first one about the installation of Tomcat and MySQL, the paragraph about the creation of schema you can unite with MySQL
- `default user(role admin)` -> `default user with admin role`
- move credentials to the next line and before `bob@gmail.com` add `email: `

Nick97-git reviewed (commented):
Code:
- Add stubs in db.properties for MySQL properties
- Look through service classes and change DataProcessingException -> RuntimeException
- DataProcessingException -> remove redundant constructor and empty line
- Look through all models classes and add annotations @Column and @JoinColumn where it needs, so your columns weren't camelCase

ShurionUA committed:
Changes implemented:
1. README.md changed.
2. Stubs added in db.properties
3. DataProcessingException changed by RunTimeException
4. Redundant empty line deleted
5. Annotations added to models classes.

Nick97-git commented:
You missed some:
Code:
- DataProcessingException -> remove redundant constructor and empty line
Readme:
- let's remove block `Requirements`
- move credentials to the next line and before bob@gmail.com add `email:`

ShurionUA committed:
Changes implemented:
1. README.md changed.
2. DataProcessingException changed by RunTimeException on Controller layer.
4. Redundant constructor deleted.
